formats/json/commonMain/src/kotlinx/serialization/json/JsonElementBuilders.kt

```diff
@@ -186,6 +195,41 @@ public fun JsonArrayBuilder.addJsonObject(builderAction: JsonObjectBuilder.() ->
 public fun JsonArrayBuilder.addJsonArray(builderAction: JsonArrayBuilder.() -> Unit): Boolean =
     add(buildJsonArray(builderAction))
 
+/**
+ * Adds the given JSON [element] and [elements] to a resulting JSON array.
+ *
+ * @return `true` if the list was changed as the result of the operation.
+ */
+public fun JsonArrayBuilder.addAll(element: JsonElement, vararg elements: JsonElement): Boolean {
```

Comment:
Why did you use `(element, elements)` structure? I do not see any possible overload resolution conflicts.

Reply:
404 reason not found. I've removed it.